client/src/graphql/current-user.query.js

```diff
@@ -26,6 +26,11 @@ export default gql`
         startTime
         endTime
         timeSegments {
+          user {
```

Comment:
This was deliberate - I starting going down this path based on our conversations the other day (when asking to optimize the queries)  - it can be moved out of the general user query in a future pull request (just wanted to get it in for the demo).

Reply:
perfect